coopterrones/color-palette
# get random palette and display in state of App.js

#### What's this PR do?
- sets the state of colors in `App.js` by calling `apiCalls.getRandomPalette()` fetch request

#### Where should the reviewer start?
- `App.js` line 5

#### How should this be manually tested?
- test by running `npm start` and using react dev tools to check that state has updated in App
- state should have an array of five elements that within themselves have an array of three numbers representing RGB values
